src/clients/meta/MetaClient.h

```diff
@@ -143,6 +144,10 @@ using IndexStatus = std::tuple<std::string, std::string, std::string>;
 using UserRolesMap = std::unordered_map<std::string, std::vector<cpp2::RoleItem>>;
 // get user password by account
 using UserPasswordMap = std::unordered_map<std::string, std::string>;
+// Mapping of user name and remaining wrong password attempts
+using UserPasswordAttemptsRemain = std::unordered_map<std::string, uint32>;
+// Mapping of user name and the timestamp when the account is locked
+using UserLoginLockTime = std::unordered_map<std::string, uint32>;
 
```

Comment:
Could these 3 map be merged to one map?
eg.
std::unordered_map<std::string, PassInfo>.

Reply:
I've considered this, but `UserPasswordMap` is used in the `localThreadCache` as a read-only container, and the rest two objects require both read and write. I believe we could keep it like this for now.